package.json

```diff
@@ -28,6 +28,7 @@
     "clean": "lerna run clean",
     "reset": "lerna run reset && lerna clean --yes && shx rm -rf node_modules/ && yarn setup"
   },
+
```

Comment:
```suggestion
```

Reply:
Okay, just not necessary.

Reply:
fixed in c9562eec904cabada1f650c950ed9ae88aacd317